src/odemis/acq/test/fastem_test.py

```diff
@@ -130,6 +131,34 @@ def test_estimateTiledAcquisitionTime(self):
         # check that estimated time increases with dwell time
         self.assertGreater(est_time_2, est_time_1)
 
+    def test_overview_acquisition_cancel(self):
+        """Test cancelling the overview image acquisition."""
+        # This should be used by the acquisition
+        self.stream.dwellTime.value = 1e-6  # s
+
+        # Known position of the center scintillator in the sample carrier coordinate system
+        scintillator5_area = (-0.007, -0.007, 0.007, 0.007)  # l, b, r, t
+
+        # Future to acquire the overview image
+        f = fastem.acquireTiledArea(self.stream, self.stage, scintillator5_area)
+        f.add_done_callback(self.on_acquisition_done)
+
+        # Wait a bit for the acquisition to happen and then cancel it
+        time.sleep(5)
+        f.cancel()
+        # Assert that the acquisition was cancelled
```

Comment:
As it runs in a separate thread, the task might take some time to cancel, and eventually call the `on_acquisition_done()`. So I'd suggest to insert here a `time.sleep(1)` to make sure the test passes on all systems.